ui-components/src/commonMain/kotlin/org/jetbrains/kotlinconf/ui/components/TalkCard.kt

```diff
@@ -413,7 +429,7 @@ internal fun TalkCardPreview() {
                 status = TalkStatus.Live,
                 onSubmitFeedbackWithComment = { e, s -> println("Feedback, emotion + comment: $e, $s") },
                 onSubmitFeedback = { e -> println("Feedback, emotion only: $e") },
-                onClick = { "Clicked session" },
+                onClick = { println("Clicked session") },
```

Comment:
please consider use logging instead

Reply:
This is just the code of the Compose preview, it's never used in production